zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -871,6 +871,10 @@ public boolean isPrometheusMetricEnabled() {
         completeConfiguration.put(c.getVarName(), sysConfig.getString(c.getVarName()));
       } else if (envConfig.containsKey(c.name())) {
         completeConfiguration.put(c.getVarName(), envConfig.getString(c.name()));
+      }else {
+        if (getString(c)!=null){
+          completeConfiguration.put(c.getVarName(),getString(c));
+        }
```

Comment:
Since `envConfig` and `sysConfig` are checked in `getString()`, would it be okay to delete lines 870~873?

However, it seems that the check order of `envConfig` and `sysConfig` is different only in `getCompleteConfiguration()`.
@zjffdu @Reamer  Which of `envConfig` and `sysConfig` takes precedence?

Reply:
@cuspymd You can find the order in the [current docs](https://github.com/apache/zeppelin/blob/master/docs/setup/operation/configuration.md).
Comment by @zjffdu: https://github.com/apache/zeppelin/pull/4028#discussion_r560194922

I noticed a small typo. PR #4123 should fix this.

Reply:
> Since `envConfig` and `sysConfig` are checked in `getString()`, would it be okay to delete lines 870~873?

You should be right. I think we can also start with an empty Hashmap.
```java
  public Map<String, String> getCompleteConfiguration() {
    Map<String, String> completeConfiguration = new HashMap<>();
    for (ConfVars c : ConfVars.values()) {
        if (getString(c) != null){
            completeConfiguration.put(c.getVarName(), getString(c));
        }
    }
    return completeConfiguration;
  }
```

Reply:
@cuspymd   You look  very carefully，i did not notice  the getString() method  
@Reamer  thank you ,I will modify according to the code  you provided